src/main/java/org/sagebionetworks/bridge/models/permissions/PermissionDetail.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.bridge.models.permissions;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.sagebionetworks.bridge.models.accounts.AccountRef;
+
+import java.util.Objects;
+
+@JsonPropertyOrder({ "guid", "userId", "accessLevel", "entityType", "entityId", "entityRef", "userAccountRef" })
+public final class PermissionDetail {
+    
+    private final String guid;
+    private final String userId;
+    private final AccessLevel accessLevel;
+    private final EntityType entityType;
+    private final String entityId;
+    private final EntityRef entityRef;
```

Comment:
Same here... entityType and entityId are part of entityRef which should be exposed under the property "entity"

Reply:
Removed entityType and entityId, changed entityRef name to "entity".